uae_src/drawing.cpp

```diff
@@ -5511,3 +5511,22 @@ int isvsync(void)
 	else
 		return isvsync_chipset ();
 }
+
+
+//////////////////////////////////////////////////////////////////////////
+namespace qd { namespace uae {
+void on_app_exit_drawing()
```

Comment:
Yep, there are definitely a few places in WinUAE where memory is not deallocated on quiting. 
I fixed it just in case when I was trying to find memory corruption through memory leaks.
The bug is not very critical, as the OS is supposed to take care of deallocated memory.
So I don't know what is better to do, put it in a separate PR  and take it to the original WinUAE rep. 
Or we can continue to ignore this memory in the quiting and leave the code as it was before?

Reply:
Yeah, the OS will clean up things, but it's also nice to have it not leak if there already is existing cleanup code I think it makes sense that everything gets cleaned up. I think we can leave it as is for now.

Reply:
OK, then I've split the UAE memory cleanup changes into a separate commit, when they start accepting PRs again we can bring it to them.